### DISCUSSION TITLE
Configuring Workflow example to use java server

### DISCUSSION THREAD

**Discussion by madamedefarge** [eclipse-glsp/glsp#1328] [General]
I am using the workflow example as a starting point for my project. I have successfully run the workflow example in glsp-examples with the embedded node server.
Now I am trying to get it running with the java server and am using Eclipse to run the glsp-server code in debug mode. I am launching the glsp-server using the Start_Workflow_Example_Server launch configuration.
Then I am running yarn start:external to run the browser-app. It connects to the server successfully.
However everytime I make an edit in the diagram I am prompted that the "source of editor 'example1.wf' has changed" ;-
<img width="769" alt="Screenshot 2024-05-02 at 10 52 00" src="https://github.com/eclipse-glsp/glsp/assets/1956357/7da6b322-23d7-4032-b2e5-633e82eb7ab3">

How can I fix this? How do I get the workflow example running with a java server please?

**Comment by tortmayr** [comment]
Hi @madamedefarge,

I tried to reproduce this issue but for me everything works as expected.
I did the following:
- Checkout and build https://github.com/eclipse-glsp/glsp-server/releases/tag/v2.0.0
- Start the server via `Start_Workflow_Example_Server` launch config
- Start the workflow example (of the glsp-examples) repository with yarn start:external

Editing the diagram etc. works as expected, no prompt of `Source of xyz changed` appears.

Could you provide me with additional details about your system? What os, java, node and maven version are you using?

**Reply by madamedefarge** [reply]
Hi @tortmayr 
I redid the build and have an issue flagged by a unit test which is likely relevant to the issue. See my post of a few days ago for the details.
Do you not get the failure in FileWatcherTest when you do a build?

**Comment by madamedefarge** [comment]
So I went back and did a rebuild from scratch. 
I checked out from here: https://github.com/eclipse-glsp/glsp-server/releases/tag/v2.0.0
and did a mvn build in the root: mvn clean verify -Pm2 -Pfatjar
and got failures in FileWatcherTest;-
[ERROR] Failures: 
[ERROR]   FileWatcherTest.changingWatchedFileNotifiesClient:88->assertNotifications:153 expected: <1> but was: <0>
[ERROR]   FileWatcherTest.deletingWatchedFileNotifiesClient:105->assertNotifications:153 expected: <1> but was: <0>

<img width="1177" alt="Screenshot 2024-05-03 at 12 48 07" src="https://github.com/eclipse-glsp/glsp/assets/1956357/16186b92-cb83-4b7a-bd80-a19d2e68b876">

This was on a macOS Sonoma 14.4 system ;-
Java  Temurin 17.0.9
Maven 3.9.6

I retried it on a Windows 11 system also with Java 17.0.9 and Maven 3.9.6 and got the same result.
